pkg/trace/api/otlp_test.go

```diff
@@ -1274,6 +1397,138 @@ func trimSpaces(str string) string {
 	return out.String()
 }
 
+func makeSpanLinkSlice(traceId, spanId, traceState string, attrs map[string]string, dropped uint32) ptrace.SpanLinkSlice {
+	s := ptrace.NewSpanLinkSlice()
+	l := s.AppendEmpty()
+	buf, err := hex.DecodeString(traceId)
+	if err != nil {
+		panic(err)
+	}
+	l.SetTraceID(*(*pcommon.TraceID)(buf))
+	buf, err = hex.DecodeString(spanId)
+	if err != nil {
+		panic(err)
```

Comment:
It might be prettier to just pass `t` and do a `t.Fatal`. A panic is a bit too much maybe?